app/views/trips/_trip.html.erb

```diff
@@ -0,0 +1,9 @@
+<tr>
+  <td><%= trip.n_girls %></td>
+  <td><%= trip.n_boys %></td>
+  <td><%= trip.n_teachers %></td>
+  <td><%= trip.n_bus_drivers %></td>
+  <td><%= trip.start_date.strftime("%d/%m/%Y") %></td>
+  <td><%= trip.end_date.strftime("%d/%m/%Y") %></td>
+  <td><%= Group.find(trip.group_id).name %></td>
```

Comment:
Use `trip.group.name`?
Another suggestion is to add this to Trip:
```
delegate :name, to: :group, prefix: true
```
Which gives us `trip.group_name` to be used in the view

Reply:
Done in `d593505`